app/javascript/components/batch_update/helpers/elasticsearch.js

```diff
@@ -128,3 +140,40 @@ const genomedMatcher = genomedFilter => {
       return null
   }
 }
+
+const NSOMatcher = NSOFilter => {
```

Comment:
This is super minor but what if we collapse these lines here to a more collapsed state like we did elsewhere? This just is super hard to read.